template/project.yml

```diff
@@ -15,21 +15,30 @@ project:
   # These are random strings used to configure the hash salt.
   hash_salt: ${project.hash_salt}
 
-# Configuration settings for new git repository.
-git:
-  default_branch: master
-  remotes:
-    # Defining git remotes allows builds deployed via CI.
-    - bolt8@svn-5223.devcloud.hosting.acquia.com:bolt8.git
-
 drush:
   aliases:
     # The remote environment from which the database will be pulled.
     remote: ${machine_name}.test
     # The local environment against which all local drush commands are run.
     local: self
 
-# Hosting environment flags.
-# Examples: acsf (Acquia Cloud Site Factory), ac (Acquia Cloud)
-# hosting: "acsf"
-# hosting: "ac"
+# Define deployment environments, and git repositories here here.
+# Defining git remotes allows builds deployed via CI.
+# Defining hosting allows certain artifacts to be deployed for different hosts
+
+deployments:
+  - acsf
+  - myhosting
+
+acsf:
+  hosting: acsf
+  gits:
+    - bolt8@svn-5223.devcloud.hosting.acquia.com:bolt8.git
+    - bolt@bolt.com:bolt8.git
+  name: Profserv ACSF
+  
+myhosting:
+  hosting: ace
```

Comment:
I'm wondering if the default branch needs to be the same for all. It's what will be created when bolt is initialized and it's the branch that will be checked out I believe when the deploy artifacts are being made.

Used in line 130 of `build.xml` in the `create` task.
```
<exec command="git checkout -b ${git.default_branch}" dir="${bolt.new.dir}" logoutput="true" passthru="true" />
```